packages/contracts/src/dollar/core/BancorPower.sol

```diff
@@ -0,0 +1,380 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * bancor formula by bancor
+ * https://github.com/bancorprotocol/contracts
+ * Modified from the original
+ * Split Power.sol out from BancorFormula.sol
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements;
+ * and to You under the Apache License, Version 2.0. "
+ */
+contract BancorPower {
```

Comment:
I don't see where this contract is used so we can remove it

Reply:
Removed